### ISSUE TITLE
Yellow focus ring doesn't meet WCAG 2.1 contrast requirements

### ISSUE THREAD

**Issue by davidakennedy** [CMSgov/mgov-design-system#15]
**Issue description**
The focus styles don't meet WCAG 2.0/2.1 contrast requirements. See the screenshot.

![Screenshot showing yellow focus ring, with red "Low 2:1 contrast" and a red arrow pointing to it.](https://user-images.githubusercontent.com/1473618/109228674-326e0880-7790-11eb-9671-73da7d812028.png)

**Remediation guidance**
The color should be changed to meet a 3:1 contrast ratio.

**Does this impact visual design?**
Yes

**Relevant standards**
1.4.11 Non-text Contrast: https://www.w3.org/TR/WCAG21/#non-text-contrast

**Additional context**
While this aims for WCAG 2.1, a standard that the Core Design System doesn't fully adhere to yet, I still wanted to file this since the contrast was rather low. I noticed this when reviewing the new Medicare Accounts experience. It's captured in the ticket: WETGA-192.
